lib/kafo/logger.rb

```diff
@@ -20,7 +20,7 @@ def error_buffer
       end
     end
 
-    PATTERN = "[%5l %d %c] %m\n"
+    PATTERN = "[%5l %d] %m\n"
```

Comment:
After comparing the before/after output in your example and reading https://www.rubydoc.info/gems/logging/Logging/Layouts/Pattern , it's still not clear to me what this change does. @ehelms could you briefly explain ?

Reply:
This drops the "name" of the logger from the output which is of no consequence to the user or a developer investigating.

Original:
```
[ INFO 2020-07-28T17:27:08 main]  /Service[smart_proxy_dynflow_core]: Starting to evaluate the resource (3066 of 3078)
[DEBUG 2020-07-28T17:27:08 main]  Executing: '/bin/systemctl is-active -- smart_proxy_dynflow_core'
[DEBUG 2020-07-28T17:27:08 main]  Executing: '/bin/systemctl is-enabled -- smart_proxy_dynflow_core'
[ INFO 2020-07-28T17:27:08 main]  /Service[smart_proxy_dynflow_core]: Evaluated in 0.10 seconds
[ INFO 2020-07-28T17:27:08 main]  Class[Foreman_proxy::Plugin::Dynflow]: Starting to evaluate the resource (3067 of 3078)
```

With this change:
```
[ INFO 2020-07-28T17:27:08 ]  /Service[smart_proxy_dynflow_core]: Starting to evaluate the resource (3066 of 3078)
[DEBUG 2020-07-28T17:27:08 ]  Executing: '/bin/systemctl is-active -- smart_proxy_dynflow_core'
[DEBUG 2020-07-28T17:27:08 ]  Executing: '/bin/systemctl is-enabled -- smart_proxy_dynflow_core'
[ INFO 2020-07-28T17:27:08 ]  /Service[smart_proxy_dynflow_core]: Evaluated in 0.10 seconds
[ INFO 2020-07-28T17:27:08 ]  Class[Foreman_proxy::Plugin::Dynflow]: Starting to evaluate the resource (3067 of 3078)
```

Reply:
Great, that looks like a nice change

Reply:
I'm not opposed, but IMHO it deserves its own change.